docs/src/main/sphinx/functions/lambda.rst

```diff
@@ -4,6 +4,8 @@
 Lambda expressions
 ==================
 
+Lambda expressions are an extension to SQL used to manipulate nested data such as arrays.
```

Comment:
Lambda expressions are more generic than manipulating nested data. For example, there is a version of `regexp_replace` that takes a lambda expression

Reply:
I've replace the statement with this one:

> Lambda expressions are anonymous functions which are passed as arguments to higher-order SQL functions.